### DISCUSSION TITLE
Kitodo Docker Repository

### DISCUSSION THREAD

**Discussion by markusweigelt** [kitodo/kitodo-production#4937] [Ideas]
How about a central repository in which we can maintain the Docker configurations and Dockerfiles for Kitodo.Production, Kitodo.Presentation etc. 

Atm there are several docker folders in the kitodo-contrib directory among other topics and implementation stuff. 

A central repository where we can maintain, document and manage the dockerfiles or docker-compose files for our Kitodo applications would be necessary in my opinion.

**Comment by stweil** [comment]
See related discussion for issue #4313.
